python/ray/tune/logger/logger.py

```diff
@@ -213,13 +213,13 @@ def represent_sequence(self, tag, sequence, flow_style=None):
 
 
 @DeveloperAPI
-def pretty_print(result):
+def pretty_print(result, blacklisted_keys: Optional[Set[str]] = None):
     result = result.copy()
     result.update(config=None)  # drop config from pretty print
     result.update(hist_stats=None)  # drop hist_stats from pretty print
     out = {}
     for k, v in result.items():
-        if v is not None:
+        if v is not None and (blacklisted_keys is None or k not in blacklisted_keys):
```

Comment:
```suggestion
        if v is not None and k not in exclude:
```

Reply:
Need to check if `exclude` is None (in which case, it's not iterable).

Also it's not good practice to have exclude taking in an optional list from function signature. and should avoid that.

Reply:
Should be fine with the above `exclude = exclude or set()`?